testsuite/tests/input/tex/Bussproofs.test.ts

```diff
@@ -310,140 +312,140 @@ describe('BussproofsRegInf', () => {
-    <mpadded height="+.5em" width="+.5em" voffset="-.15em" semantics="bspr_prooflabel:left">
-      <mstyle displaystyle="false" scriptlevel="0">
-        <mtext>L</mtext>
-      </mstyle>
-    </mpadded>
-    <mtable align="top 2" rowlines="solid" framespacing="0 0" semantics="bspr_inferenceRule:down">
-      <mtr>
-        <mtd>
-          <mtable framespacing="0 0">
-            <mtr>
-              <mtd rowalign="bottom">
-                <mrow data-latex="\\LeftLabel{L}" semantics="bspr_axiom:true">
-                  <mspace width=".5ex"></mspace>
-                  <mstyle displaystyle="false" scriptlevel="0">
-                    <mtext>A</mtext>
-                  </mstyle>
-                  <mspace width=".5ex"></mspace>
-                </mrow>
-              </mtd>
-            </mtr>
-          </mtable>
-        </mtd>
-      </mtr>
-      <mtr>
-        <mtd>
-          <mrow>
-            <mspace width=".5ex"></mspace>
-            <mstyle displaystyle="false" scriptlevel="0">
-              <mtext>B</mtext>
-            </mstyle>
-            <mspace width=".5ex"></mspace>
-          </mrow>
-        </mtd>
-      </mtr>
-    </mtable>
-  </mrow>
-</math>`
+      `<math xmlns=\"http://www.w3.org/1998/Math/MathML\" data-latex=\"\\begin{prooftree}\\AxiomC{A}\\LeftLabel{L}\\UnaryInfC{B}\\end{prooftree}\" display=\"block\">
+      <mrow data-latex=\"\\begin{prooftree}\\AxiomC{A}\\LeftLabel{L}\\UnaryInfC{B}\\end{prooftree}\" data-latex-item=\"{prooftree}\" semantics=\"bspr_labelledRule:left;bspr_inference:1;bspr_proof:true\">
+        <mpadded height=\".25em\" depth=\"+.25em\" width=\"+.5ex\" voffset=\"-.25em\" semantics=\"bspr_prooflabel:left\">
+          <mstyle displaystyle=\"false\" scriptlevel=\"0\">
+            <mtext>L</mtext>
+          </mstyle>
+        </mpadded>
+        <mtable align=\"top 2\" rowlines=\"solid\" framespacing=\"0 0\" semantics=\"bspr_inferenceRule:down\">
+          <mtr>
+            <mtd>
+              <mtable framespacing=\"0 0\">
+                <mtr>
+                  <mtd rowalign=\"bottom\">
+                    <mrow data-latex=\"\\LeftLabel{L}\" semantics=\"bspr_axiom:true\">
+                      <mspace width=\".5ex\"></mspace>
+                      <mstyle displaystyle=\"false\" scriptlevel=\"0\">
+                        <mtext>A</mtext>
+                      </mstyle>
+                      <mspace width=\".5ex\"></mspace>
+                    </mrow>
+                  </mtd>
+                </mtr>
+              </mtable>
+            </mtd>
+          </mtr>
+          <mtr>
+            <mtd>
+              <mrow>
+                <mspace width=\".5ex\"></mspace>
+                <mstyle displaystyle=\"false\" scriptlevel=\"0\">
+                  <mtext>B</mtext>
+                </mstyle>
+                <mspace width=\".5ex\"></mspace>
+              </mrow>
+            </mtd>
+          </mtr>
+        </mtable>
+      </mrow>
+    </math>`
```

Comment:
Are all the backslashes needed for the quotation marks?  I guess they don't hurt, but they make the expected output a little harder to read.

Reply:
That comes from the little Emacs function that allows for copying received values from the test log.
And they actually do hurt, at least in the last three tests, where I had to manually copy things over from the test output as the slashed quotes ruined the tests.

I'll remove them and then merge.